build.gradle

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'java'
+    id 'co.riiid.gradle' version '0.4.2'
+}
+
+apply from: 'plugin-helpers.gradle'
+
+project.ext.pluginVersion = '1.4'
+project.ext.fullVersion = project.git.distVersion() ? "${project.pluginVersion}-${project.git.distVersion()}" : project.pluginVersion
+
+group = 'com.tw.go.plugin'
+version = project.fullVersion
+
+project.ext.pluginDesc = [
+        repo       : rootProject.name,
+        version    : project.version,
+        goCdVersion: '17.3.0',
+        vendorName : 'GoCD Contributors',
+        vendorUrl  : 'https://github.com/gocd/kubernetes-elastic-agents'
+]
+
+repositories {
+    mavenCentral()
+}
+
+project(':gerrit-cs-status') {
```

Comment:
It will be better if we can move it to the respective module.

Reply:
Done.